src/maxKey.js

```diff
@@ -9,11 +9,25 @@
  * @returns {number} The key with the largest value
  */
 function maxKey(obj) {
-  // This is your job. :)
+  let currentLargestValue = 0;
```

Comment:
Look at the example in the comment above this function. This function is meant to return the _key_ which has the largest value, not the largest value itself. That's why the function is called `maxKey` (and why the example is what it is).

Heck, if you're returning the largest value itself just do...

```javascript
let max = require('./max');

function maxValue(obj) {
  return max(Object.values(obj));
}
```